src/main/java/gregtech/common/metatileentities/electric/multiblockpart/MetaTileEntityMultiblockPart.java

```diff
@@ -25,6 +25,7 @@
     private final int tier;
     private BlockPos controllerPos;
     private MultiblockControllerBase controllerTile;
+    private ICubeRenderer hatchColor = null;
```

Comment:
The name should probably be changed to something more descriptive. hatchColor seems like it might be confusing in relation to paining/tinting TEs.